synapse/util/batching_queue.py

```diff
@@ -38,6 +39,24 @@
 V = TypeVar("V")
 R = TypeVar("R")
 
+number_queued = Gauge(
+    "synapse_util_batching_queue_number_queued",
+    "The number of items waiting in the queue across all keys",
+    labelnames=("name",),
+)
+
+number_in_flight = Gauge(
+    "synapse_util_batching_queue_in_flight",
```

Comment:
To me, `in_flight` would be the ones actually being processed right now

```suggestion
    "synapse_util_batching_queue_number_pending",
```

Reply:
That is a better name.